### PR TITLE
🔧 use STORYBOOK_PERCY_ENV for yarn snapshot

### PR DESCRIPTION
**Pull request by bdolly**
This is a flag that is used to only render Percy specifc markup when storybook is built for percy consumption. This helps performance in the main storbook build

This is just a quick fix, a better solution in the future would be a custom storybook decorator that takes a param with the test markup and checks the env vars

### CODE REVIEW

**Comment by dankolbman** [comment]
UIKit Storybook for Review

Built with commit 7111971fd210f7974ba5e0953b3de1c317716c41

https://deploy-preview-127--kf-uikit.netlify.com

File: .storybook/config.js [hunk 1]
```diff
@@ -5,14 +5,14 @@ import { withA11y } from '@storybook/addon-a11y';
 import { withInfo } from '@storybook/addon-info';
 import centered from '@storybook/addon-centered';
 import '../src/tailwind.src.css';
-import gridImg from '../static/spacing_grid__4px_cells.png'
-
+import gridImg from '../static/spacing_grid__4px_cells.png';
+console.log(process.env);
```
Comment (dankolbman) [inline .storybook/config.js]: 🔥 Probably don't need this

**Comment by bdolly** [comment]
ok if you checkout the deploy preview and search the source for `show-in-tests` it shouldn't show up but all the proper snapshots are in percy

File: src/components/Grid/Grid.story.jsx [hunk 2]
```diff
@@ -29,45 +29,47 @@ stories.add(
...
-              </div>
-            ))}
-            {range(1, 13).map(() => (
-              <div className="cell-1 row-2 h-12 text-black text-center border bg-white">
-                <p className="mt-0 pt-12 text-xs">60px</p>
-              </div>
-            ))}
-          </GridContainer>
+        {process.env.STORYBOOK_PERCY_ENV ? (
+          <section className="show-in-tests" style={{ width: 1280 }}>
```
Comment (dankolbman) [inline src/components/Grid/Grid.story.jsx]: Do we still need the `show-in-tests` if we are using the conditional on the env?

Comment (bdolly) [reply src/components/Grid/Grid.story.jsx]: not necessarily, but can't hurt as a fail safe

**Pull request merged**